python_docs_theme/static/sidebar.js_t

```diff
@@ -44,46 +44,43 @@ const initialiseSidebar = () => {
  #}
 {% if sphinx_version_tuple is defined and sphinx_version_tuple[0] >= 5 %}
   const sidebarButton = document.getElementById("sidebarbutton")
-  const sidebarArrow = sidebarButton.querySelector('span')
 {% else %}
   // create the sidebar button element
   const sidebarButton = document.createElement("div")
   sidebarButton.id = "sidebarbutton"
-  // create the sidebar button arrow element
-  const sidebarArrow = document.createElement("span")
-  sidebarArrow.innerText =  "«"
-  sidebarButton.appendChild(sidebarArrow)
-  sidebar.appendChild(sidebarButton)
 {% endif %}
+  sidebarbutton.innerHTML = ""
+  sidebarbutton.tabindex = "0"  // make it focusable
+  sidebarbutton.role = "slider"
+  sidebarbutton.title = _("Resize sidebar")
+  sidebarbutton.setAttribute("aria-label", _("Resize sidebar by dragging"))
+  sidebarbutton.setAttribute("aria-valuetext", _("Sidebar width XXX pixels"))
```

Comment:
I didn't manage to find `ngettext`, is it available?

Reply:
`_` is an alias for `Documentation.gettext` in Sphinx's `doctools.js`:

https://github.com/mne-tools/mne-tools.github.io/blob/1c89caf568b03d2571239053ad05b1a084f6252a/dev/_static/doctools.js#L146-L147

Which also contains `Documentation.ngettext`, you could try that?